src/pytorch_lightning/strategies/launchers/subprocess_script.py

```diff
@@ -23,7 +23,9 @@
 import pytorch_lightning as pl
 from pytorch_lightning.plugins.environments.cluster_environment import ClusterEnvironment
 from pytorch_lightning.strategies.launchers.base import _Launcher
-from pytorch_lightning.utilities import _HYDRA_AVAILABLE
+from pytorch_lightning.utilities.imports import _RequirementAvailable
+
+_HYDRA_AVAILABLE = _RequirementAvailable("hydra")
 
 if _HYDRA_AVAILABLE:
```

Comment:
I suggest moving these to local imports in the code. Otherwise we don't reap the benefits of https://github.com/Lightning-AI/lightning/issues/12786 because we will try to import the module on import.